### PR TITLE
Docker log limit and missing preview  size env variable

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

- Docker log limit
- Missing env vars

## Test Plan

Existing

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Review by eldadfux — COMMENTED** [review]
Should we have the same in our dev compose file for consistency? @TorstenDittmann can we come up with a shorter syntax?

File: app/views/install/compose.phtml [hunk 1]
```diff
@@ -36,6 +36,11 @@ services:
   appwrite:
     image: <?php echo $organization; ?>/<?php echo $image; ?>:<?php echo $version."\n"; ?>
     container_name: appwrite
+    logging:
```
Comment (TorstenDittmann) [inline app/views/install/compose.phtml:39]: Considering our docker compose file is already 500+ lines, we might wanna use some YML features to clean things up.

```yml
x-logging: &x-logging
    driver: 'json-file'
    options:
          max-file: 5
          max-size: 10m

services:
  appwrite:
    container_name: appwrite
    logging:
      <<: *x-logging
```
something like this should work

Comment (TorstenDittmann) [reply app/views/install/compose.phtml:39]: For reference, I really wanna push for something like this => https://gist.github.com/TorstenDittmann/7dfd1c10345023066f1b98098426fb26

I might be creating a PR today :-)

**Comment by TorstenDittmann** [comment]
Also, what's the reasoning behind the addition of logging parameters?

**Comment by eldadfux** [comment]
> Also, what's the reasoning behind the addition of logging parameters?

Limiting the log size. In big setups the worker logs become very heavy!

File: docker-compose.yml [hunk 2]
```diff
@@ -1,3 +1,9 @@
+x-logging: &x-logging
+  logging:
+    driver: 'json-file'
+    options:
+      max-file: 5
+      max-size: 10m
 # WARNING!
```
Comment (eldadfux) [inline docker-compose.yml:7]: Let's keep the warning on top.

**Review by eldadfux — APPROVED** [review]

**Pull request merged**